README.md

```diff
@@ -1,5 +1,5 @@
-## Awesome Books App
-A To Do List app built with Javascript and Webpack
```

Comment:
- [ ] Please change the header of your README file as it refers to an older project.
I think you should consider giving your README more time to organize it as it is one of the important parts of your project and in the future, it will reflect in how you are attentive to details.